service/controller/azuremachinepool/handler/nodepool/template/main.json

```diff
@@ -229,7 +229,7 @@
         "singlePlacementGroup": "[parameters('vmssSinglePlacementGroup')]",
         "virtualMachineProfile": {
           "evictionPolicy": "[if(parameters('spotInstancesEnabled'), 'Delete', json('null'))]",
-          "priority": "[if(parameters('spotInstancesEnabled'), 'Spot', 'Regular')]",
```

Comment:
Before spot instances we didn't specify this field so it was empty.
The field is not editable, so we have to keep it as "" for non-spot instance VMSS